app/code/Magento/Backend/etc/adminhtml/system.xml

```diff
@@ -311,34 +311,6 @@
             <label>System</label>
             <tab>advanced</tab>
             <resource>Magento_Config::config_system</resource>
-            <group id="smtp" translate="label" type="text" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
I moved them from the default backend to the Email module because of the references to the Email Source. I can also just add the new options there if that's preferred.